index.html

```diff
@@ -122,9 +122,9 @@ <h3 class="header-3">
                     </h3>    
                 </div>
                 <form action="https://formspree.io/f/xrgraoke" method="post" class="form">
-                    <input name="name" type="text" class="entry" id="name" placeholder="Full Name" maxlength="30" data-name='full-name' required><br>
-                    <input name="email" type="email" class="entry" id="email" placeholder="Please input your email" data-email='email' required><br>
-                    <textarea name="comment" id="comment" maxlength="500" cols="30" rows="10" class="entry" placeholder="Type in something" required></textarea><br>    
+                    <input name="name" type="text" class="entry" id="name" placeholder="Full Name" maxlength="50" data-name='full-name'><br>
+                    <input name="email" type="text" class="entry" id="email" data-email='email' required><br>
+                    <textarea name="comment" id="comment" maxlength="600" cols="30" rows="10" class="entry" placeholder="Type in something" required></textarea><br>    
                     <button type="submit" class="btn" id="get-in-touch">Get in touch</button><span id="message"></span>
                 </form>
             </article>
```

Comment:
- [ ] [OPTIONAL]  Kindly change from `article` to `section` for  improved and matching semantics to best improve SEO of your portfolio website